src/worldstate/routine/include/State.h

```diff
@@ -0,0 +1,32 @@
+//
+// Created by joel on 06/05/18.
+//
+
+#ifndef PROJECT_ROUTINE_H
+#define PROJECT_ROUTINE_H
+
+#include <ros/ros.h>
+#include <worldstate/state_msg.h>
+/*
+ * Documentation to Provide
+ *
+ * Communication Class: {Starting from this state, it is possible to return
```

Comment:
Not quite sure what's meant by `Communication Class` here?

Reply:
I don't really understand this comment either. Is it just to list the state that can transition to this state?

Reply:
In this case, I believe a communication class is the same thing as a routine. 

In a finite state machine or Markov Chain, a communicating class is a set of all nodes wherein if you start at one, it's possible to return by traversing through any combination of the other ones. Like lookingForGate, aligningWithGate, and passingGate all belong to the same C.C. since it's possible to choose one and return to it. 

I'll change this so that it just asks to enumerate which states it is possible to traverse to.